src/algebra/hom/iterate.lean

```diff
@@ -3,9 +3,7 @@ Copyright (c) 2020 Yury Kudryashov. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Yury Kudryashov
 -/
-
-import logic.function.iterate
-import group_theory.perm.basic
+import algebra.group_power.lemmas
```

Comment:
`algebra.hom.iterate` and `group_theory.perm.basic` are neighbor files. It makes more sense to have `equiv.perm.coe_pow` in `group_theory.perm.basic` and I have an upcoming PR that will make it refl, so `algebra.hom.iterate` won't even be needed anymore!